pkg/controllers/securitypolicy/namespace_watcher.go

```diff
@@ -53,21 +53,21 @@ func (e *EnqueueRequestForNamespace) Update(updateEvent event.UpdateEvent, l wor
 		return
 	}
 
-	shouldReconcile := false
+	shouldReconcile1 := false
```

Comment:
Why to change the name?

Reply:
It's changed by accident. Revert change. Thanks for advice.